### DISCUSSION TITLE
hao4k 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#13] [模板讨论]
hao4k 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by hecarli555** [comment]
求大佬攻破了，，，，出问题了
Failed at 2/5 request,
Fail assert: {"re": "未定义操作", "from": "content"} from failed_asserts,
Request URL: https://www.hao4k.cn/plugin.php?id=k_misign%3Asign&operation=qiandao&formhash={{formhash|urlencode}}&from=insign&inajax=1&ajaxtarget=JD_sign

**Comment by hecarli555** [comment]
大佬有人没？

**Reply by wjf0214** [reply]
更新了。再试下

**Reply by hecarli555** [reply]
删除旧的模板，然后重建缓存后重新订阅新的模板还是不行，提示这个
<html>
<body>


Failed at 2/5 request,Fail assert: {"re": "未定义操作", "from": "content"} from failed_asserts,Request URL: https://www.hao4k.cn/plugin.php?id=k_misign%3Asign&operation=qiandao&formhash={{formhash\|urlencode}}&from=insign&inajax=1&ajaxtarget=JD_sign
--



</body>
</html>

**Reply by wjf0214** [reply]
重建缓存还是有缓存的。你要清除cdn的缓存，但是麻烦。要不你就先从github上下。要不就等明天缓存更新后再订阅。

**Reply by hecarli555** [reply]
大佬牛逼！新版能正常使用了，感谢大佬！

**Reply by hecarli555** [reply]
太贱了，，，，他们站今天又换了验证方式，换成看图输入正确英文了。。。
![image](https://github.com/wjf0214/qd-templates/assets/76683744/1d31e029-4fb2-48c0-8b34-0b4653640237)
<html>
<body>


Failed at 13/16 request,Fail assert: {"re": "未定义操作", "from": "content"} from failed_asserts,Request URL: https://www.hao4k.cn/plugin.php?id=k_misign%3Asign&operation=qiandao&formhash={{formhash\|urlencode}}&from=insign&inajax=1&ajaxtarget=JD_sign
--



</body>
</html>

**Reply by mtyzaq** [reply]
就是，刚更新2天，又换验证方式，又不能用了

**Reply by wjf0214** [reply]
更新了，再试试。如果提示验证码错误，多试几次。验证码识别有可能不是很准确。

**Reply by mtyzaq** [reply]
hao4K疯了！！

**Reply by hecarli555** [reply]
更新了…提示阻断…看来他们知道签到项目了…
Failed at 1/22 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.hao4k.cn/qiandao/

**Comment by hecarli555** [comment]
大佬…新版模板提示403…这怎么破？

**Comment by hecarli555** [comment]
大佬大佬。。。这个模板是不修了么5555

**Comment by hecarli555** [comment]
跪求大佬修复！感谢

**Reply by wjf0214** [reply]
最近忙，而且刚开始那几天网站一直在变，抽空我看看。

**Reply by wjf0214** [reply]
最近没怎么看，现在hao4k的防护是稳定下来了吗？是一种还是轮着来的。

**Reply by wjf0214** [reply]
按今天看到的先调整了下，目前我测试可以。有问题我再抽空调整吧。

**Reply by hecarli555** [reply]
感谢大佬百忙当中抽时间来修复，，，，感谢

**Reply by hecarli555** [reply]
服气，，，，好了一天又坏了。。。。
<html>
<body>


败 | Failed at 18/26 request,Fail assert: {"re": "注册", "from": "content"} from failed_asserts,Request URL: https://www.hao4k.cn{{tz_url}}
-- | --



</body>
</html>

**Comment by KoWming** [comment]
![QQ截图20230825173740](https://github.com/wjf0214/qd-templates/assets/96684377/960a579c-d679-4b43-afc8-7520bba819f5)
今天测试OK。

**Reply by hecarli555** [reply]
你ck一天就失效吗？

**Comment by hecarli555** [comment]
> ![QQ截图20230825173740](https://user-images.githubusercontent.com/96684377/263231876-960a579c-d679-4b43-afc8-7520bba819f5.png) 今天测试OK。

刚又重新订阅模板重新抓ck又好了，，，奇奇怪怪的，继续观察一下

**Comment by hecarli555** [comment]
貌似ck一天就失效。。。。？能不能出个账号密码版的？

**Comment by hecarli555** [comment]
难受，，，ck一天就失效了

**Reply by wjf0214** [reply]
没啊，CK一直都有效。。。

**Reply by hecarli555** [reply]
为啥我提示的这个，，，
<html>
<body>


Failed at 18/26 request,Fail assert: {"re": "注册", "from": "content"} from failed_asserts,Request URL: https://www.hao4k.cn{{tz_url}}
--



</body>
</html>

**Reply by hecarli555** [reply]
需要重新抓ck才能正常。。。。，然后第二天又会出现上面的提示，，，

**Reply by wjf0214** [reply]
没记错的话，网站单点登录，你签到后，网页登陆应该掉了，你要是登陆，签到ck就失效。

**Reply by hecarli555** [reply]
那。。。。咋搞？

**Reply by xuexijiaoben** [reply]
我的好像也是一直掉，奇怪，我试试获取cookie后不再登录试试

**Reply by hecarli555** [reply]
登录网站，获取ck后登出网站，ck第二天还是会失效。。。。

**Comment by miaoxiaozhao** [comment]
我也遇到第二天失效的问题了。

**Reply by wjf0214** [reply]
有没有登录过4ksj，我记得之前做这2个模板的时候好像会互相有影响。挤掉登录。登录后不退出，不操作其他的看看。。

**Reply by hecarli555** [reply]
4ksj新模板没有问题了一直正常，就hao4k签到完就掉

**Comment by hecarli555** [comment]
新版hao4k要改为只能app签到了，，，，大佬有计划破解一下么？

**Reply by wjf0214** [reply]
后面有空看下。

**Reply by hecarli555** [reply]
大佬有空修复一下么？

**Comment by hecarli555** [comment]
请问hao4k的模板还修复么？

**Reply by wjf0214** [reply]
暂时不修复，现在只能抓App的，App估计有一些参数验证。这些参数怎么生成就需要研究了。不处理的话，签到是能签到，但是没几天号会被ban。后续再看吧。
